public/app/core/selectors/navModel.ts

```diff
@@ -43,7 +43,7 @@ function getSectionRoot(node: NavModelItem): NavModelItem {
   if (root.children) {
     root.children = root.children.map((item) => {
       if (item.id === node.id) {
-        return { ...node, active: true };
+        return { ...item, active: true };
```

Comment:
This should break a unit test, as item is missing the updated children prop (with active set)

Reply:
If I remember correctly

Reply:
yeah - this is basically a draft for now. i want to get the e2e test reenabled first 😄